incubator/hnc/internal/validators/namespace.go

```diff
@@ -99,6 +106,16 @@ func (v *Namespace) handle(req *nsRequest) admission.Response {
 	return allow("")
 }
 
+func (v *Namespace) illegalExcludedNamespaceLabel(req *nsRequest) admission.Response {
+	for l := range req.ns.Labels {
+		if l == api.LabelExcludedNamespace && !config.ExcludedNamespaces[req.ns.Name] {
+			msg := fmt.Sprintf("You cannot exclude this namespace using the %q label. See https://github.com/kubernetes-sigs/multi-tenancy/blob/master/incubator/hnc/docs/user-guide/concepts.md#hncx-k8sioexcluded-namespace-label-on-namespaces for detail.", api.LabelExcludedNamespace)
```

Comment:
That HTML anchor (everything after the '#') looks brittle and likely to change. Maybe put an explicit anchor into the file and refer to that?